config.yml

```diff
@@ -33,6 +33,7 @@ defaults:
     relengAPIProxy:
       image: 'taskcluster/relengapi-proxy:2.3.1'
       token: !env RELENG_API_TOKEN
+    disableSeccomp: false
```

Comment:
I think I was not clear in my review. What I meant was a configuration that allows the worker to process `disableSeccomp: true` feature. Look at the `allowPrivileged` configuration for an example on how to do so.

Reply:
Sorry, I still don't understand. The example config you linked to is for `allowPrivileged: false`, so where should I look for an example config where `allowPrivileged` is true?

I modeled the change after `allowPtrace` which is a closer analogue, and I don't see that referenced in config.yml.

Reply:
The role of `allowPrivileged` is adding another layer of security to the worker-pool. It is a worker-pool configuration that when it is `true`, [it doesn't allow task to run as a privileged container, even if it requests in the payload](https://github.com/taskcluster/docker-worker/blob/master/src/lib/task.js#L327-L329). Maybe it is easier if you implement `disableSeccomp` as a capability instead of a feature.

Reply:
Thanks, I understand now. I'll take a look.